domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/AcpFAQService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017 the original author or authors.
+ *
+ * This file is part of jBB Application Project.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.frontend.api.faq;
+
+import org.jbb.frontend.api.faq.model.AcpFAQTuple;
+
+import java.util.List;
+
+public interface AcpFAQService {
```

Comment:
They are connected.  
FAQ as a <K,V> pair is display to e2e user however FAQ settings are located in ACP bookmark. 
Only Administrator can define new or delete FAQ question ... so ye, it's connected